library/foundation/src/main/java/com/epmedu/animeal/foundation/input/TextInputField.kt

```diff
@@ -0,0 +1,97 @@
+package com.epmedu.animeal.foundation.input
+
+import android.content.res.Configuration.UI_MODE_NIGHT_YES
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.material.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.epmedu.animeal.base.theme.AnimealTheme
+import com.epmedu.animeal.base.theme.CustomColor.CursorColor
+import com.epmedu.animeal.base.theme.CustomColor.DarkerGrey
+import com.epmedu.animeal.base.theme.CustomColor.LynxWhite
+
+@Composable
+fun TextInputField(
+    modifier: Modifier,
```

Comment:
I guess will be better

`modifier: Modifier = Modifier`

Reply:
Rigth now it's require to pass modifier

Reply:
done